pytest/tests/loadtest/locust/common/inscription.py

```diff
@@ -0,0 +1,109 @@
+import logging
+from concurrent import futures
+import random
+import string
+import sys
+import pathlib
+import typing
+from locust import events, runners
+import common.base as base
+
+sys.path.append(str(pathlib.Path(__file__).resolve().parents[4] / 'lib'))
+
+import key
+from common.base import Account, Deploy, NearNodeProxy, NearUser, FunctionCall, INIT_DONE
+from configured_logger import new_logger
+
+logger = new_logger(level=logging.INFO)
+
+
+class MintInscription(FunctionCall):
+
+    def __init__(self, contract_account_id: str, sender: Account, tick, amt):
+        # Attach exactly 1 yoctoNEAR according to NEP-141 to avoid calls from restricted access keys
+        super().__init__(sender, contract_account_id, "inscribe", balance=0)
+        self.sender = sender
+        self.tick = tick
+        self.amt = amt
+
+    def args(self) -> dict:
+        return {
+            "p": "nrc-20",
+            "op": "mint",
+            "tick": self.tick,
+            "amt": str(int(self.amt))
+        }
+
+    def sender_account(self) -> Account:
+        return self.sender
+
+
+class DeployInscription(FunctionCall):
```

Comment:
Nit: Given that this is almost the same code as above, you can factor it out into `InscriptionOp` and add `op` as an argument to instantiate these individual call types (either directly at call sites, or by still keeping the dedicated classes for this)

Reply:
Conceptually those are two different actions, so I feel like they should be in two separate classes.
I could add a parent claass `InsctiptionOp` and inherit from it, but I'm not a fan of that approach, it feels too complex for its own good.
Also the json arguments sometimes are different for deploy actions - deploy actions accept arguments like maximum supply, maximum amount per mint and stuff like that. Having a separate class allows to add other json args to the deploy action in the future.

I'd prefer to KISS as is.